src/Enum.php

```diff
@@ -373,6 +380,28 @@ protected static function getLocalizedDescription($value): ?string
         return null;
     }
 
+    /**
+     * Get the description of a value from its PHP attribute.
+     *
+     * @param  mixed  $value
+     * @return string|null
+     */
+    protected static function getAttributeDescription($value): ?string
+    {
+        $reflection = self::getReflection();
+        $constantName = static::getKey($value);
+        $constReflection = $reflection->getReflectionConstant($constantName);
+        $descriptionAttributes = $constReflection->getAttributes(Description::class);
+
+        if (count($descriptionAttributes)) {
```

Comment:
```suggestion
        if (count($descriptionAttributes) === 1) {
```

I think we should add a check below that throws if the attribute is used more than once.

Reply:
From the PHP docs:
`By default an attribute can only be used once per declaration. If the attribute should be repeatable on declarations it must be specified as part of the bitmask to the #[Attribute] declaration.`
https://www.php.net/manual/en/language.attributes.classes.php

So this is already the case. I have however added a more friendly exception message instead of throwing an error.

Reply:
Looks good. One minor nit: let's move this test below the check for `count() === 1` in order to have a fast path for correct implementations.

Reply:
Yep good point